include/oneapi/dpl/internal/gcd_impl.h

```diff
@@ -55,6 +55,11 @@ __get_abs(_Source __t)
 } // namespace internal
 
 // gcd
+// We need have own implementation instead of std::gcd because
```

Comment:
Is not it better to make one comment at the beginning of the header for both `gcd` and `lcm`?

Reply:
Fixed